src/stores/connectors/keys/keys.go

```diff
@@ -35,5 +35,9 @@ func isSupportedAlgo(alg *entities.Algorithm) bool {
 		return true
 	}
 
+	if alg.Type == entities.Eddsa && alg.EllipticCurve == entities.X25519 {
```

Comment:
how come this function exists? Shouldn't we have specific functions for each store?